pom.xml

```diff
@@ -36,6 +36,11 @@
   </scm>
 
   <dependencies>
+    <dependency>
+      <groupId>org.jvnet.hudson.plugins</groupId>
+      <artifactId>extended-read-permission</artifactId>
+      <version>3.2</version>
```

Comment:
Do I understand correctly that since 2.222 this would not be needed.

Reply:
Yes it will not be needed